debian/pbspro-server.install

```diff
@@ -0,0 +1,308 @@
+etc/pbs_db_schema.sql /opt/pbs/etc
```

Comment:
Thank you @subhasisb, that explains it. I didn't realize it was a barrier to OBS.